pkgs/development/interpreters/erlang/R19.nix

```diff
@@ -45,6 +45,8 @@ stdenv.mkDerivation rec {
 
   preConfigure = ''
     ./otp_build autoconf
+  '' + optionalString stdenv.isDarwin ''
+    substituteInPlace configure --replace "sw_vers" "/usr/bin/sw_vers"
```

Comment:
@LnL7 this is true; how does the other system then work with this tool?
I'm also curious to see how the build system in Erlang 19 has changed to use this tool. Is this tool even necessary for the compilation step?
Can we try manually linking all .frameworks necessary into the build expression, and then remove the call to this tool by `sbustituteInPlace`?